apicapi/apic_manager.py

```diff
@@ -218,12 +218,16 @@ def ensure_infra_created_on_apic(self):
         setup profiles for switches, modules and ports
         """
 
+        for domain in self.domains:
+            if (self.provision_infra or
+                getattr(domain, 'vmm_type', None) in
```

Comment:
Should we just create any kind of VMM?

`if [...] or isinstance(domain, apic_domain.VmDomain)):`